src/Modules/DashboardModule/components/Dashboard/Dashboard.tsx

```diff
@@ -1,7 +1,195 @@
+import GroupWorkIcon from '@mui/icons-material/GroupWork';
+import AdsClickIcon from '@mui/icons-material/AdsClick';
+import { Stack } from "@mui/material";
+
+ChartJS.register(ArcElement, Tooltip, Legend);
+const Item = styled(Paper)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === "dark" ? "#1A2027" : "#fff",
+  ...theme.typography.body2,
+  padding: theme.spacing(5),
+  textAlign: "center",
+  background: "#1a1b1e",
+  color: theme.palette.text.secondary,
+}));
+
+interface datadashboard {
+  success: boolean;
+  message: string;
+  data: {
+    rooms: number;
+    facilities: number;
+    bookings: {
+      pending: number;
+      completed: number;
+    };
+    ads: number;
+    users: {
+      user: number;
+      admin: number;
+    };
+
+  };
+
+}
 
 export default function Dashboard() {
+  const { requestHeaders }: any = useContext(AuthContext);
+  const [dashlist, setDashList] = useState<datadashboard | null>(null);
+
+  const getListOfRooms = async () => {
+    try {
+      const response = await axios.get(
+        "https://upskilling-egypt.com:3000/api/v0/admin/dashboard",
+        {
+          headers: requestHeaders,
+        }
+      );
+
+      setDashList(response.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  useEffect(() => {
+    getListOfRooms();
+  }, []);
+  const roomNumber = dashlist?.data.rooms;
+  const facilitiesNumber = dashlist?.data.facilities;
+  const ADSNumber = dashlist?.data.ads;
+  //Variables for Bookings Chart
+  const pendingNumber = dashlist?.data.bookings.pending;
+  const completedNumber = dashlist?.data.bookings.completed;
+
+  const userNumber = dashlist?.data.users.user;
+  const AdminNumber = dashlist?.data.users.admin;
+
+  const data = {
+    labels: [`Users:${userNumber}`, `Admins:${AdminNumber}`],
+    datasets: [
+      {
+        label: "Count",
+        data: [userNumber, AdminNumber],
+        backgroundColor: ["#42A5F5", "#66BB6A"],
+        hoverBackgroundColor: ["#64B5F6", "#81C784"],
+      },
+    ],
+  };
+
+  const options = {
+    responsive: true,
+    plugins: {
+      legend: {
+        position: "bottom",
+      },
+      title: {
+        display: true,
+        text: "Users and Admins",
+      },
+    },
+  };
+
+
+  const booking = {
+    labels: [
+      'Pending', 'Completed'
+    ],
+    datasets: [{
+      label: 'count',
+      data: [pendingNumber, completedNumber],
+      backgroundColor: [
+        'rgb(255, 99, 132)',
+        'rgb(54, 162, 235)',
+        'rgb(255, 205, 86)'
+      ],
+
+
+      hoverOffset: 4
+    }]
+  };
+  const stylesItem = {
+    display: "flex", color: "#fff", justifyContent: "space-between", fontSize: "17px", marginBottom:
+      "50px"
+  }
   return (
-    <div></div>
-  )
+    <>
+      <Box sx={{ flexGrow: 1, mt: 5 }}>
+        <Grid container spacing={7}>
+          <Grid item xs={12} md={4}>
+            <Item style={stylesItem}>
+              <div>
+                <div className="title-rooms">
```

Comment:
It seems that we are using global classes. This is not a good practice. We should stick to MUI's design system regarding colors, spacing and so on. We can change this to a box and include the styles in the sx props or use the emotion style package used with MUI to provide styles to the component we want.

Reply:
this was just a mock class ,we removed it at requested